wherehows-dao/build.gradle

```diff
@@ -4,4 +4,7 @@ dependencies {
   compile externalDependency.commons_lang3
   compile externalDependency.slf4j_api
   compile externalDependency.spring_jdbc
+  compile externalDependency.jackson_databind
```

Comment:
In one of the models Dataset.java there is one field , see below
public JsonNode properties;
which requires jackson dependency , hence i have included them